src/call_wrapper.ts

```diff
@@ -83,11 +85,15 @@ class CallWrapper {
 
         const hasArrayParameters = parameters.some((parameter) => parameter instanceof Array);
 
-        if (hasArrayParameters) {
-            const memory = new WebAssembly.Memory({ initial: CallWrapper.calculateInitialMemorySize(parameters) });
+        if (hasArrayParameters || this.arrayLiteralMemorySize !== 0) {
+            const memoryDescriptor = {
```

Comment:
Change the name `memoryDescriptor` to `initialMemory`

Reply:
I named it `memoryDescriptor` becaus that is the name of the parameter name of `WebAssembly.Memory()`. I think it's a fitting name.